test/functional/index.test.js

```diff
@@ -1350,4 +1354,96 @@ describe('Indexes', function() {
       });
     }
   });
+
+  context('commitQuorum', function() {
+    function throwErrorTest(testCommand) {
+      return {
+        metadata: { requires: { mongodb: '<4.4' } },
+        test: function() {
+          return withClient(
+            this.configuration.newClient(),
+            client =>
+              new Promise(resolve => {
```

Comment:
:facepalm: remind me to make `withClient` work with just callbacks..

Reply:
I took a closer look at the implementation and I think it actually does!

Reply:
:metal: